activerecord/lib/active_record/result.rb

```diff
@@ -89,17 +89,17 @@ def to_h
       alias_method :to_hash, :to_h
     end
 
-    attr_reader :columns, :rows
+    attr_reader :columns, :rows, :affected_rows
```

Comment:
The documentation at the top of this file needs to be updated to include affected_rows.

Reply:
Added some docs for `affected_rows` 👍